test/views/DraftMovementViewSpec.scala

```diff
@@ -60,6 +61,38 @@ class DraftMovementViewSpec extends SpecBase with ViewBehaviours with MovementSu
           ))
         }
 
+        "for a validation failure scenario" - {
+
+          val validationFailures: Seq[MovementValidationFailure] = Seq(
+            //scalastyle:off magic.number
+            MovementValidationFailure(12, "This is an error."),
+            MovementValidationFailure(13, "This is an error."),
+            MovementValidationFailure(14, "This is an error.")
+            //scalastyle:on magic.number
+          )
+
+          implicit val msgs: Messages = messages(Seq(messagesForLanguage.lang))
```

Comment:
Could this implicit be put on line 45? Then it can remove duplication across these 3 tests

Reply:
Sure 🙂